api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -3956,6 +3956,17 @@ message CertificateCreate {
     (gogoproto.embed) = true,
     (gogoproto.jsontag) = ""
   ];
+
+  // CertificateAuthority holds information about creator of certificate
+  CertificateAuthority CertificateAuthority = 5 [(gogoproto.jsontag) = "certificate_authority"];
+}
+
+// CertificateAuthority holds information about creator of certificate
+message CertificateAuthority {
+  // ID is identifier of the cert authority (type and domain)
+  string ID = 1 [(gogoproto.jsontag) = "id,omitempty"];
```

Comment:
Actually, instead of combining these fields, why not have a field for each (type and domain)?